packages/features/ee/organizations/components/OrganizationMemberAvatar.tsx

```diff
@@ -0,0 +1,47 @@
+import classNames from "@calcom/lib/classNames";
```

Comment:
Renamed `OrganizationAvatar` to `OrganizationMemberAvatar` to more correctly convey what it is
Also, it started using UserAvatar instead of Avatar